services/api/src/messaging/players.service.ts

```diff
@@ -186,6 +240,19 @@ export const playerConnected: Handler = (
     sendMessage(ws, MessageType.SetMyTurn, myTurn);
   }
 
+  sendMessage(
+    ws,
+    MessageType.UpdateGameState,
+    gameRoomService.getGameState(ws.roomId),
+  );
+
+  if (gameRoomService.getGameState(ws.roomId) === GameState.GAME_NOT_STARTED) {
+    if (gameService.getVoters(ws.roomId).length > 1) {
+      gameRoomService.setGameState(ws.roomId, GameState.GAME_IN_PROGRESS);
+      publishGameState(ws);
```

Comment:
Could we move this call into a single place and it would be executed after every action taken by players?

Reply:
Why should we send so many messages? For example if user makes two turns(for example changes his mind), we would publish game that has not changed state twice. I think we should leave game state publishing, to when its really needed or game state changes